### PR TITLE
release instructions

### PR DESCRIPTION
**Pull request by grosser**
@AshleyTaylorZendesk looks right ?

### CODE REVIEW

**Comment by AshleyTaylorZendesk** [comment]
> @AshleyTaylorZendesk looks right ?

Mostly the only thing that isn't right is you have an environment protection rule to only run on branches matching `v*` since you are merging to `master` this won't trigger a workflow run

**Comment by grosser** [comment]
ah yes it should be `git push --tags`

**Comment by grosser** [comment]
can you +1 the PR ?

**Review by AshleyTaylorZendesk — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -272,3 +271,16 @@ curl -I 'http://localhost/api/users/1.json' -X POST -d '{ "user": { "id": 1 } }'
...
+
+## Development
+
+### Releasing a new version
+
+```
+git checkout master && git fetch origin && git reset --hard origin/master
+bundle exec rake bump:<patch|minor|major>
+git tag v<tag>
+git push --tags
```
Comment (bquorning) [inline README.md:283]: I would recommend using `rake release` instead. Just set `ENV['gem_push'] = 'false'` in the Rakefile. See https://github.com/zendesk/active_record_host_pool/commit/59eb7c0a23308c789311fadcc06b315733dc92a5 for inspiration.

**Pull request merged**